content/200-concepts/100-components/01-prisma-schema/06-relations/410-referential-actions/index.mdx

```diff
@@ -150,16 +150,18 @@ The following caveats apply:
 
 The following table shows which referential action each database supports.
 
-| Database    | Cascade | Restrict | NoAction | SetNull | SetDefault |
-| :---------- | :------ | :------- | :------- | :------ | :--------- |
-| PostgreSQL  | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
-| MySQL       | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**†    |
-| SQLite      | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
-| SQL Server  | **✔️**  | **❌**‡  | **✔️**   | **✔️**  | **✔️**     |
-| CockroachDB | **✔️**  | **✔️**   | **✔️**   | **✔️**  | **✔️**     |
```

Comment:
Is this table supposed to have had Cockroach removed?

Reply:
No, that was a merge fuckup from my side I think.

Reply:
Fixed